### PR TITLE
Add /telemetry redirect

### PR DESCRIPTION
**Pull request by kylegach**
- Add footer link

### CODE REVIEW

**Review by winkerVSbecks — APPROVED** [review]

File: site-metadata.js [hunk 1]
```diff
@@ -113,6 +114,7 @@ siteMetadata.urls.navCommunityLinks = [
   { title: 'Get involved', href: siteMetadata.urls.community, isGatsby: true },
   { title: 'Use cases', href: siteMetadata.urls.useCases, isGatsby: true },
   { title: 'Support', href: siteMetadata.urls.support, isGatsby: true },
+  { title: 'Telemetry', href: siteMetadata.urls.telemetry, isGatsby: true },
```
Comment (kylegach) [inline site-metadata.js:117]: Adding this here will add the link to both the Community nav menu in the Header and the Storybook section of the Footer. Preview here: https://deploy-preview-359--storybook-frontpage.netlify.app/

Is that desired? /cc @winkerVSbecks

Comment (winkerVSbecks) [reply site-metadata.js:117]: How about adding it to the footer directly (not via the site metadata). So either it won't show up in tutorials or blog for now or you could patch those footers too. I'm fine with either of those options.

Comment (kylegach) [reply site-metadata.js]: Done.

- https://github.com/chromaui/blog/pull/89
- https://github.com/chromaui/learnstorybook.com/pull/596

Comment (winkerVSbecks) [reply site-metadata.js]: Thanks!

**Review by shilman — COMMENTED** [review]
@kylegach @winkerVSbecks Does this need to redirect to the 6.5 docs between now and the time 6.5 goes live?

**Comment by kylegach** [comment]
> @kylegach @winkerVSbecks Does this need to redirect to the 6.5 docs between now and the time 6.5 goes live?

Good point. Imo, yes, it should.

**Pull request merged**